### PR TITLE
Automate grafana tunnel

### PR DESCRIPTION
**Pull request by gonzaloserrano**
Just a small toy created while trying to understand the codebase better.

### CODE REVIEW

File: modules/addons/grafana/main.tf [hunk 1]
```diff
@@ -47,3 +48,9 @@ resource "kubectl_manifest" "tsb_dashboards" {
   for_each   = var.dashboards
   yaml_body  = data.kubectl_path_documents.tsb_dashboards[each.key].documents[0]
 }
+
+resource "local_file" "ssh_tunnel" {
```
Comment (gonzaloserrano) [inline modules/addons/grafana/main.tf:52]: self-review: now that I think this should be called port-forward instead of ssh-tunnel 🤔

**Comment by nacx** [comment]
JFYI in case you haven't seen it, you can set these two values in your root `.tfvars` file:
```json
"grafana_service_type": "LoadBalancer",
"argocd_service_type": "LoadBalancer",
```
and those services will have a public IP allocated so that you don't need to port-forward.

**Comment by smarunich** [comment]
Please lets limit the number of custom tools/overlays such as and lets use the service type instead as @nacx mentioned, for the scripts like that lets not overload the tf.

Basically the repo is to be exposed to the customer and my chapter is to keep it as simple as possible, so lets keep the deployment patterns as production ones and limit the extras that can be addressed by the existing functionality. 

I do appreciate though having the PRs like that to be opened, as it gives a chance to spread the knowledge and intentions behind the repo.

**Review by smarunich — CHANGES_REQUESTED** [review]
--

**Comment by gonzaloserrano** [comment]
Fair enough! 👍

**Pull request closed without merging**